docs/with-props.md

```diff
@@ -0,0 +1,11 @@
+## Usage with recompose's withProps
+
+You can pass additional props to your components using recompose's `withProps` higher-order component. The `withProps` api is laid out [here](https://github.com/acdlite/recompose/blob/master/docs/API.md#withprops).
```

Comment:
"here" isn't a very accessible anchor link - perhaps it could be "in the recompose documentation" instead?